### PR TITLE
Fix #522 (needed bc of PEP 479)

### LINKED ISSUE #522: For loop in twython search results raises StopIteration
Hello,

I'm very new at python and I'm trying to search the results of my twitter query.

When I do, once the end of the generator is reached, my entire code stops from a runtime error raised by a StopIteration. 

Upon researching, this is a new thing in Python 3.7 (which is what I'm using for this project) where instead of just continuing on once a StopIteration has been met, it stops the entire script from running.

So my question is, are there any ways to either provide a list of how many results there are or a way to exception out the StopIteration from presenting itself so I can continue running my code?

Thank you in advance

### PR DESCRIPTION
**Pull request by zseri**
Fixes #522.

### CODE REVIEW

File: twython/api.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 dealing with the Twitter API
 """
 
+from __future__ import generator_stop
```
Comment (zserik) [inline twython/api.py:12]: I'm unsure if this `import` is actually needed.

**Comment by zseri** [comment]
@RaitoBezarius could you please review this PR?

@ryanmcgrath could you please merge this PR? (refs: https://github.com/ryanmcgrath/twython/issues/522#issuecomment-582153066, https://github.com/ryanmcgrath/twython/issues/522#issuecomment-554648143)

**Comment by ryanmcgrath** [comment]
Ah, I can merge this if one or two people test it and say it's good to go. I assume this'd also need a release bump?

**Comment by zseri** [comment]
I'm not completely sure anymore, but I thought I had tested it.

**Comment by ryanmcgrath** [comment]
Oh, I believe you tested it - it's more that I don't touch this part of the library these days, so want to be sure that:

A) it's got a decent pair of eyes on it (though given issue 522 I suppose we could just yolo it, since the discussion there has been around for some time)

B) we do a version bump so we're not blowing up too many existing setups etc

Let's ping @RaitoBezarius one more time, and then if they're too busy to take a peek (given it's also been some time) we'll go the yolo approach.

**Comment by RaitoBezarius** [comment]
To be honest, I didn't test it yet, but it looks good, I think the yolo plan is not that much absurd.
I think the `__future__` import makes sense for older Python 3 versions, but should work more or less fine on latest ones IMHO.

**Comment by ryanmcgrath** [comment]
Alright, a `3.9.0` should be on Pypi now.

**Pull request merged**